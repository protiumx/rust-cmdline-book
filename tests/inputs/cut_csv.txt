Title,Description,Pages
Soso,"Soso was, Soso was",900
La Malicia,Not much to say,777
La Hojarrasca,Somewhere in Colombia,356
