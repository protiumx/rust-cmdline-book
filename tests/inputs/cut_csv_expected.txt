Description
"Soso was, Soso was"
Not much to say
Somewhere in Colombia
